src/python/grpcio/grpc/_cython/cygrpc.pyx

```diff
@@ -81,6 +81,7 @@ include "_cygrpc/aio/server.pyx.pxi"
 # initialize gRPC
 #
 cdef _initialize():
+  InitializeLog()
```

Comment:
Do we know that `_initialize` is called exactly once even if client code imports grpc multiple times? If not, we should have an internal state to check if we have already initialized it.